src/lib/types/next-auth.d.ts

```
@@ -0,0 +1,35 @@
+  interface User {
+    accessToken: string;
+    user: {
+        "_id": string,
+        "firstName": string,
+        "lastName": string,
+        "email": string,
+        "gender": string,
+        "phone": string,
+        "photo": string,
+        "role": string,
+        "wishlist": any[],
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)